tests/backends/test_utils.py

```diff
@@ -13,3 +14,14 @@ def test_load_backend_invalid_name(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg) as cm:
             load_backend('foo')
         self.assertEqual(str(cm.exception.__cause__), "No module named 'foo'")
+
+    def test_truncate_name(self):
+        self.assertEqual(truncate_name('some_table', 10), 'some_table')
```

Comment:
I added basic non-regression tests in separate PR https://github.com/django/django/pull/8072.